src/services/iiif-parser.js

```diff
@@ -324,13 +325,24 @@ export function getSegmentMap({ manifest }) {
 /**
  * Get poster image for video resources
  * @param {Object} manifest
+ * @param {Number} canvasIndex
  */
-export function getPoster(manifest) {
-  if (!parseManifest(manifest).getThumbnail()) {
+export function getPoster(manifest, canvasIndex) {
+  let posterUrl;
+  let placeholderCanvas = parseManifest(manifest)
+    .getSequences()[0]
+    .getCanvasByIndex(canvasIndex)
+    .__jsonld['placeholderCanvas'];
+  if (placeholderCanvas) {
+    let annotations = placeholderCanvas['items'];
+    let items = parseAnnotations(annotations, 'painting');
+    if (items.length > 0) {
+      posterUrl = items[0].getBody()[0].id;
+    }
```

Comment:
If ramp assumes that the poster is a static image then it might be good to do a check here that the annotation body is type `Image` since placeholderCanvas can have any content type usually allowed in canvases.

Future work might be having ramp respect the width and height of placeholderCanvas since it can be different than the canvas itself.